travvy/src/components/Onboarding.vue

```diff
@@ -29,20 +34,38 @@
     <button class="nextActivity">Family</button>   <button class="nextActivity">Historical</button>   <button class="nextActivity">Active</button>  <button class="nextActivity">Nature</button>  <button class="nextActivity"> Entertainment</button> <button class="nextActivity">Food</button> 
     <br>
     <br>
+    <br>
+    <label>preferred_activity_type?: </label>
+    <select class="info" v-model="preferred_activity_type">
+        <option></option>
+        <option>Family</option>
+        <option>Historical</option>
+        <option>Active</option>
+        <option>Nature</option>
+        <option>Entertainment</option>
+        <option>Food</option>
```

Comment:
This information will need to be taken into account when recommending attractions as a ticket for next sprint

Reply:
Yup, next sprint we can add this functionality to our Search feature for the AttractionsList page